contribute-to-tidb/review-a-pr-stage.md

```diff
@@ -0,0 +1,235 @@
+# How to Review a Pull Request
+
+This guide is for contributors who want to contribute to the code review. Thank you for your effort. A good review of code is crucial for an open source project.
+
+ This guide is intended to help both the contributors and reviewers to ensure the following:
+
+* The review keeps the high quality of TiDB code.
+* Reviewers cover all important aspects of a contribution
+* Contributors have a good contribution experience..
+* The review avoids situations in which contributors and reviewers spend a lot of time refining a contribution that gets rejected later.
+
+
+## Things to know before the code review
+
+Why is code review so important?
+
+* A good review can eliminate bugs early.
+* A good review can make the project maintainable in the long run.
+
+To conduct a good review, it is always helpful to read the following high-level guidelines about code review in the TiDB community:
+
+### Code review is more important than writing your own PR
+
+Nowadays software development is no longer a single person’s piece of art but the result of team cooperation. To speed up the development and make the processing scale, it is important to avoid blocking. If all developers in an organization prioritize writing code over reviewing code, more and more PRs waiting for code review get blocked, and the whole team slows down.
+
+### It's the PR author's responsibility to get the PR merged
+
+We had a fierce debate on how to speed up the review process long long ago. What the role the reviewer and PR owner should play and what is each of their responsibility to get the PR merged faster.
+
+After the meeting, most of the participants agree with that the PR owner is responsible for getting PR merged.
```

Comment:
```suggestion
Because PR authors are the ones who want to merge the PRs into the repository the most, after an offline discussion we agree on the following:
```